server/channels/api4/access_control_test.go

```diff
@@ -873,3 +873,46 @@ func TestSearchChannelsForAccessControlPolicy(t *testing.T) {
 		CheckForbiddenStatus(t, resp)
 	})
 }
+
+func TestCreateAccessControlSyncJob_RateLimit(t *testing.T) {
```

Comment:
A nice to have would be to test that we are actually checking the permissions.

Other things that are not tested (but probably are also not tested in other tests) is error handling, and whether things gets correctly in the audit or not.